test/infrastructure/docker/Makefile

```diff
@@ -69,7 +69,11 @@ help:  ## Display this help
 ## --------------------------------------
 
 .PHONY: test
-test: ## Run tests
+test: ## Run tests.
```

Comment:
Might be a little bit out of the scope of this PR, but for consistency, I suggest to add support for TEST_ARGS here as well.

Reply:
+1

Reply:
Added the env var for now.